service-monitor/src/main/java/com/yahoo/vespa/service/monitor/internal/health/HealthMonitor.java

```diff
@@ -63,11 +71,17 @@ public void close() {
         healthClient.close();
     }
 
-    private long initialDelayInSeconds(long maxInitialDelayInSeconds) {
+    private long initialDelayInMillis(long maxInitialDelayInSeconds) {
         return random.nextLong() % maxInitialDelayInSeconds;
```

Comment:
This will produce negative initial delay (effectively 0) half the time

Reply:
Nice - fixed.